apps/server/src/config/index.ts

```diff
@@ -30,7 +30,11 @@ AWS.config.update({
     region: AWS_REGION,
 })
 
-const firebaseServiceAccount = path.resolve(__dirname, '../../firebase-dev-adminsdk.json')
+const firebaseServiceAccount = path.resolve(
+    __dirname,
+    '../../',
+    process.env.NODE_ENV === 'production' ? 'firebase-prod-adminsdk.json' : 'firebase-dev-adminsdk.json',
+)
```

Comment:
어차피 1개 파일만 사용 가능해서 저는 그냥 하나로 통일시키고 docker compose 에서 파일 볼륨을 잘 관리하는 방식이 좋을거 같아요
```suggestion
const firebaseServiceAccount = path.resolve(__dirname, '../../firebase-adminsdk.json')
```